example/server.js

```diff
@@ -13,7 +13,7 @@ app.listen({
   port: process.env.PORT || 8080
 }, function() {
   const host = this.address();
-  const address = `http://${host.address}:${host.port}`
+  const address = `http://${host.address}:${host.port}/index.html`
```

Comment:
since it took me some time to figure out why the instructions in the docs didn't work, I changed the url here to include index.html and guarantee the page will load

Reply:
It works just fine for me. When I type`npm run example:serve` it opens a new browser window with the example page. What was the problem exactly?

Reply:
It didn't open the page for me. I'm on windows 10 using chrome

Reply:
Page opens on http://127.0.0.1:8080/
There the html loads as `Cannot GET /`

Reply:
I'm on windows 10 using chrome

Reply:
On http://127.0.0.1:8080/index.html the example page loads

Reply:
Does the new url work for you?

Reply:
Works flawlessly both ways but I'm under a linux build. Anyway, I'll accept the PR and see what's happening later.